compile/create_binaries/create_app.jl

```diff
@@ -22,3 +22,22 @@ create_app(
 
 include("add_metadata.jl")
 add_metadata(project_dir, license_file, output_dir, git_repo)
+
+# On Windows, write ribasim.cmd in the output_dir, that starts ribasim.exe.
+# Since the bin dir contains a julia.exe and many DLLs that you may not want in your path,
+# with this script you can put output_dir in your path instead.
+if Sys.iswindows()
+    cmd = raw"""
+    @echo off
+    setlocal
```

Comment:
Does Julia cut off the whitespace, or is it now also indented in the script?

Reply:
It cuts it off, also the first newline.

Reply:
Oh nice.